sycl/test-e2e/FilterSelector/filter_list_cpu_gpu_acc.cpp

```diff
@@ -10,14 +10,14 @@
 
 // RUN: %clangxx -fsycl %S/Inputs/filter_list_queries.cpp -o %t.out
 
-// RUN: env ONEAPI_DEVICE_SELECTOR="*:acc" %t.out | FileCheck %s --check-prefixes=CHECK-ACC-ONLY
-// RUN: env ONEAPI_DEVICE_SELECTOR="*:gpu" %t.out | FileCheck %s --check-prefixes=CHECK-GPU-ONLY
-// RUN: env ONEAPI_DEVICE_SELECTOR="*:cpu" %t.out | FileCheck %s --check-prefixes=CHECK-CPU-ONLY
+// RUN: env ONEAPI_DEVICE_SELECTOR="*:acc" %{run-unfiltered-devices} %t.out | FileCheck %s --check-prefixes=CHECK-ACC-ONLY
```

Comment:
Does `%{run-unfiltered-devices}` just a nop to be stylistically accurate and to prevent adding `%{run}` here? Or there is something more?

Reply:
There is, see

https://github.com/intel/llvm/blob/daea2ba1f47f7e20c085dcc7a9d33bf5d3657445/sycl/test-e2e/format.py#L128-L133